### ISSUE TITLE
running yolo5s on kaggle doesn't work

### ISSUE THREAD

**Issue by litaldavar** [ultralytics/yolov5#3231]
Hello 

I'm trying to run the yolo5s net on my custom dataset.
I've downloaded the net and got all my files in order 
when I run the net I get errors

![yolo5s_error](https://user-images.githubusercontent.com/4908360/118784071-5b3a0180-b898-11eb-93b5-e33e78d929fe.PNG)

please help

**Comment by litaldavar** [comment]
all my code is in : https://www.kaggle.com/litaldavar/mask-detection-with-yolov5/edit

Please tell me what is missing

thanks

**Comment by glenn-jocher** [comment]
@litaldavar your kaggle URL returns a 404.

The error is due to your custom yaml. See the Train Custom Data tutorial for directions on exactly how to create your dataset yaml:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by litaldavar** [comment]
Ok this problem was solved 
but now I get a different problem
![image](https://user-images.githubusercontent.com/4908360/118819508-13c66c00-b8be-11eb-96e9-7918906bf663.png)

**Comment by glenn-jocher** [comment]
@litaldavar you can run the W&B login cell above the train cell before training, or if the problem doesn't go away you can disable W&B with the bash command `wandb disabled` before training.
